### PR TITLE
Redux

### PR DESCRIPTION
**Pull request by AlvinCYLo**
### Overview

Added placeholder redux into project for future use

TODO: Left redux bindings out since we're not too sure what business logic it will be supporting. Opted not to use react-redux template from CRA (ie project from --template redux). Will need to hook it up with the components when we decide to actually use redux

### CODE REVIEW

File: client/src/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
+import notationReducer from './reducers/notationReducer';
+import thunk from 'redux-thunk';
```
Comment (valarao) [inline client/src/redux/store.js:3]: I believe you have to run ` yarn add redux-thunk`

File: client/src/redux/actions/notationActions.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { GET_NOTATION_POSITION } from '../types';
+
+export const getNotationPosition = (notation) => (dispatch) => {
```
Comment (valarao) [inline client/src/redux/actions/notationActions.js:4]: Maybe add a TODO just so were sure where to make the changes

**Review by valarao — CHANGES_REQUESTED** [review]
Left a few comments, looks good overall!

File: client/src/index.js [hunk 3]
```diff
@@ -2,11 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import { store } from './app/store';
```
Comment (valarao) [inline client/src/index.js]: Import here isn't correct

**Review by valarao — APPROVED** [review]
LGTM!

File: client/src/App.js [hunk 4]
```diff
@@ -15,11 +17,15 @@ const useStyles = makeStyles({
 const App = () => {
   const classes = useStyles();
   return (
-    <Box className={classes.root}>
-      <Navbar />
-      <DashboardPage />
-    </Box>
+    <div classname='App'>
```
Comment (valarao) [inline client/src/App.js]: I believe this div is redundant though

Comment (AlvinCYLo) [reply client/src/App.js]: yeah, agreed

**Pull request merged**